### PR TITLE
Added a readme

### PR DESCRIPTION
**Pull request by alkemann**

### CODE REVIEW

**Comment by jimbofreedman** [comment]
I came up with
```
brew install php72 composer npm webpack wepback-dev-server
git clone git@github.com:lars-raaum/kom-inn.git
cd kom-inn
pushd admin-api
composer install
popd
pushd admin
npm install
popd
pushd web-api
composer install
popd
pushd web
npm install
popd
bash dev.sh
```
last night. I think it is everything?

**Comment by alkemann** [comment]
@jimbofreedman added install script to repo

File: bin/install.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $DIR/..
+
+brew install php72 composer npm webpack wepback-dev-server
```
Comment (sgulseth) [inline bin/install.sh]: imo global dependencies shouldn't be installed here.

Comment (alkemann) [reply bin/install.sh]: yea we can move the line to the readme?

**Pull request merged**